code/game/area/areas/ruins/space.dm

```diff
@@ -543,3 +543,29 @@
 	name = "Syndicate Laboratory 4071 Surrounding Area"
 	icon_state = "dk_yellow"
 	color = "#26773a88"
+
+// Nanotrasen Office
```

Comment:
You could cut down on areas here or use generics that are appropriate for your planet type.

Reply:
I'll fix this soon, anything I should change aside from the passthroughs?